src/exampleTypes/TermsStatsTable.js

```diff
@@ -29,7 +33,24 @@ let TermsStatsTable = injectTreeNode(
           {...{
             ...props,
             children: criteria
-              ? [
+              ? _.compact([
+                  checkable ? (
```

Comment:
Lets make a CheckableTermsStatsTable like we did for results so this component stays simpler